css-color/Overview.bs

```diff
@@ -1714,7 +1717,7 @@ Profiled, Device-dependent Colors</h2>
 	or an RGB colorspace (such as ProPhoto <!-- ref -->, <!-- would be good to mention AdobeRGB 1998 here too, need to check trademark and copyright issues first --> widely used by photographers), or any other color or monochrome output device which has been characterized.
 	In addition, for convenience,
 	CSS provides two predefined RGB color spaces:
-	DCI P3 [[!DCI-P3]],
+	p3d65 [[!DCI-P3]],
```

Comment:
@cabanier My primary comment is that the name should not be "p3d65" since it refers to something else already. A secondary question is whether the group intends to document specific display capabilities and/or encoding colorspaces. For example, sRGB is currently the encoding colorspace for the web even though not all displays can necessarily display sRGB completely.